cmd/snap-confine/seccomp-support.c

```diff
@@ -36,20 +36,7 @@
 #include "../libsnap-confine-private/string-utils.h"
 #include "../libsnap-confine-private/utils.h"
 
-#ifndef SECCOMP_FILTER_FLAG_LOG
```

Comment:
Pardon my ignorance, why does this move into a new file? I mean, it seems seccomp-support.c is an appropriate place for this code. Whats the advantage of the new  `-ext.h`  file? If we rename, maybe to `-filter.h`?

Reply:
It moved along with the place where it is needed.

I used a new file to avoid the crippled `intent` based formatter. New files use `clang-format` which produces much nicer output.